subworkflows/exomeseq-gatk4-01-preprocessing.cwl

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+requirements:
+  - class: ScatterFeatureRequirement
+  - $import: ../types/bespin-types.yml
+inputs:
+  # NOTE: How long is this expected to take?
```

Comment:
There are several question comments like this one in this file.

Reply:
Will clean those up. Thanks!